structs.go

```diff
@@ -165,6 +165,16 @@ type CreateRequest struct {
 	Flags int32
 }
 
+type CreateContainerRequest CreateRequest
+
+type CreateTTLRequest struct {
+	Path  string
+	Data  []byte
+	Acl   []ACL
+	Flags int32
+	Ttl   int64
```

Comment:
here it would now be important to comment on the unit of time. Especially since this is an exported type.

Reply:
ok